### PR TITLE
(Changes made) Added React JS Playlists in Tamil.

### PR DESCRIPTION
**Pull request by yogan-gopi**
## What does this PR do?
Added 4 Tamil playlists in React JS, a frontend framework to build UI. 

### Description
Added 4 Tamil courses in React JS by JVL code, Tamil Coding Wizard, MaanavaN Learn Code, VJ TECHNO WIZARD YouTube channels which are considered to be the best among Tamil courses.

### Why is this valuable (or not)?
I saw that there were no courses related to React in Tamil, hence I added these few hand-picked courses.

### How do we know it's really free?
All I added were YouTube videos, hence they are absolutely free.

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by eshellman** [comment]
we don't accept shortened links

**Comment by yogan-gopi** [comment]
@eshellman Made the necessary changes (Changed all the links to full links)

**Comment by eshellman** [comment]
For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes

File: courses/free-courses-ta.md [hunk 1]
```diff
@@ -93,6 +94,14 @@ CSE Tamila by Eezytutorials
 * [R Tutorial in Tamil](https://youtube.com/playlist?list=PL4unWLKFsZfeGbK28rfPDeDDD_OJGjMCC) - Tutor Joe's Stanley
 
 
+### React
+
+* [React](https://youtube.com/playlist?list=PL7BQ4lqtgECTVwBbEjQ63FPx76WYDbiwh&si=PxoLxQoXVCqi1zav) - JVL code
+* [React Basics Tamil](https://youtube.com/playlist?list=PLQeZxRj52I-HntAkC29CgxGRT9Z_-oa91&si=oe9UoqzeaUDYyoy6) - Tamil Coding Wizard
+* [React JS Tamil Tutorial for Beginners](https://youtube.com/playlist?list=PLfD4W8QfMd5DbFccLzRFeG0QjWWHGTT3-&si=X3CgUFk3PxeqA8YD) - MaanavaN Learn Code
+* [React JS Tutorial Tamil](https://youtube.com/playlist?list=PLtMr2pEysMV7DdPChnkF9Mmgdya1uR8sQ&si=ZNop81SRBf9eTGvK) - VJ TECHNO WIZARD
```
Comment (eshellman) [inline courses/free-courses-ta.md:102]:
```suggestion
* [React JS Tutorial Tamil](https://youtube.com/playlist?list=PLtMr2pEysMV7DdPChnkF9Mmgdya1uR8sQ&si=ZNop81SRBf9eTGvK) - VJ Techno Wizard
```

**Comment by yogan-gopi** [comment]
@eshellman made the required changes.

**Comment by eshellman** [comment]
thanks for sticking with this

**Pull request merged**